coda_v2_python_client/firebase_client_wrapper.py

```diff
@@ -627,6 +627,40 @@ def set_user_ids(self, dataset_id, user_ids):
         batch.commit()
         log.debug(f"Wrote {len(user_ids)} users to dataset {dataset_id}")
 
+    def get_sequence_number(self, dataset_id):
```

Comment:
Can this be 
```suggestion
    def get_highest_sequence_number(self, dataset_id):
```
instead?

Then you're returning a simple fact about the database as it is now and users can simply add one where they need to. Or, call this `get_next_available_sequence_number(self, dataset_id)` to make it clear how you're getting to this sequence number. 
(with the name `get_sequence_number`, I'd expect a message or message id to be one of the args).